src/priyanka_Kamble/Assignment_16/FrequencyOfChar.java

```diff
@@ -0,0 +1,46 @@
+package priyanka_Kamble.Assignment_16;
+
+/*
+ * Assignment - 16 : 9th April'2021
+ * 
+Find the frequency of character from a given String array.
+Note: Take String and character from the user using scanner class.
+
+Input : technocredits
+char : e
+ */
+import java.util.Scanner;
+
+public class FrequencyOfChar {
+	static String str;
+	static char ch;
```

Comment:
You should write this static/instance variable with in method as, it’s useful for only that method.  If variable going to use multiple places then it makes senses to define that variable at static/instance. just trying to explain good coding practices.